tests/modules/gtdbtk/classifywf/main.nf

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { GTDBTK_CLASSIFYWF } from '../../../../modules/gtdbtk/classifywf/main.nf' addParams( options: [:] )
+
+workflow test_gtdbtk_classifywf {
+    
+    input = [ [ id:'test', single_end:false ], // meta map
+              file(params.test_data['sarscov2']['illumina']['test_paired_end_bam'], checkIfExists: true) ]
+
+    GTDBTK_CLASSIFYWF ( input )
+}
```

Comment:
And this is the complicate part, I am not sure how to test this module since it's requires the presence of a sizeable taxonomic database on disk.

Reply:
I think for this lets do stubs, and we can verify locally

Reply:
Cool, this makes sense 👍